examples/README.md

```diff
@@ -60,3 +60,10 @@ the following positions in the latter file:
 - [JS] [signed-txn](./signed-txn) - demonstrates loading a serialized transaction from a file. A user can create a transaction and sign it (using `goal`) and send it to someone else to execute it in `algob`.
 - [TS] [htlc](./htlc-pyteal-ts) - Hash-Time-Lock-Contract Example using PyTeal.
   In this project we are transpiling the files in `js` and then using them with `algob`.
+
+### Tests Description
```

Comment:
we should update the heading to something else. Maybe "Few interesting integration tests"?

Reply:
```suggestion
### Tests Description
```
```suggestion
### Interesting Test Suites
```